cmd/juju/interact/pollster_test.go

```diff
@@ -263,6 +335,58 @@ func (PollsterSuite) TestQueryStringSchema(c *gc.C) {
 	c.Assert(w.String(), jc.Contains, "Enter region:")
 }
 
+func (PollsterSuite) TestQueryStringSchemaWithDefault(c *gc.C) {
+	schema := &jsonschema.Schema{
+		Singular: "region",
+		Type:     []jsonschema.Type{jsonschema.StringType},
+		Default:  "foo",
+	}
+	r := strings.NewReader("\n")
+	w := &bytes.Buffer{}
+	p := New(r, w, w)
+	v, err := p.QuerySchema(schema)
+	c.Assert(err, jc.ErrorIsNil)
+	s, ok := v.(string)
+	c.Check(ok, jc.IsTrue)
+	c.Check(s, gc.Equals, "foo")
+	c.Assert(w.String(), jc.Contains, "Enter region [foo]:")
+}
+
+func (PollsterSuite) TestQueryStringSchemaWithUnusedDefault(c *gc.C) {
+	schema := &jsonschema.Schema{
+		Singular: "region",
+		Type:     []jsonschema.Type{jsonschema.StringType},
+		Default:  "foo",
+	}
+	r := strings.NewReader("bar\n")
+	w := &bytes.Buffer{}
+	p := New(r, w, w)
+	v, err := p.QuerySchema(schema)
+	c.Assert(err, jc.ErrorIsNil)
+	s, ok := v.(string)
+	c.Check(ok, jc.IsTrue)
+	c.Check(s, gc.Equals, "bar")
+	c.Assert(w.String(), jc.Contains, "Enter region [foo]:")
+}
+
+func (PollsterSuite) TestQueryStringSchemaWithPromptDefault(c *gc.C) {
+	schema := &jsonschema.Schema{
+		Singular:      "region",
+		Type:          []jsonschema.Type{jsonschema.StringType},
+		Default:       "foo",
+		PromptDefault: "not foo",
+	}
+	r := strings.NewReader("\n")
+	w := &bytes.Buffer{}
+	p := New(r, w, w)
+	v, err := p.QuerySchema(schema)
+	c.Assert(err, jc.ErrorIsNil)
+	s, ok := v.(string)
+	c.Check(ok, jc.IsTrue)
+	c.Check(s, gc.Equals, "foo")
+	c.Assert(w.String(), jc.Contains, "Enter region [not foo]:")
```

Comment:
Anything that should not preclude tests from continuing -- even if they are currently the last line in the function -- should utilize `c.Check`.